package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "alphagov-wordpress-theme",
+  "version": "1.0.0",
+  "description": "Alpha Gov WordPress Theme",
+  "author": "govwordpress@gov.bc.ca",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "WordPress",
+    "Block Theme",
+    "BCGov",
+    "AlphaGov",
+    "Gutenberg"
+  ],
+  "homepage": "https://github.com/bcgov/alphagov-wordpress-theme#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bcgov/alphagov-wordpress-theme.git"
+  },
+  "bugs": {
+    "url": "https://github.com/bcgov/alphagov-wordpress-theme#readme",
+    "email": "govwordpress@gov.bc.ca"
+  },
+  "engines": {
+    "npm": ">= 10.7.0",
+    "node": ">=20.15.1"
+  },
+  "main": "index.js",
+  "dependencies": {
+    "@wordpress/icons": "^10.3.0"
+  },
+  "devDependencies": {
+    "@bcgov/bc-sans": "^2.1.0",
+    "@bcgov/wordpress-eslint": "git+https://apps.itsm.gov.bc.ca/bitbucket/scm/wp/bcgov-wordpress-eslint.git#semver:1.x",
+    "@bcgov/wordpress-stylelintrc": "git+https://apps.itsm.gov.bc.ca/bitbucket/scm/wp/bcgov-wordpress-stylelintrc.git#semver:1.x",
+    "@wordpress/scripts": "^28.3.0"
+  },
+  "scripts": {
+    "build": "wp-scripts build src/scripts/public/font.js --output-path=dist",
+    "build:production": "npm ci && npm run fix:js && npm run lint:js && npm run fix:css && npm run lint:css && npm run lint:md:docs && npm run test:unit && npm run build",
+    "lint:css": "wp-scripts lint-style",
+    "fix:css": "wp-scripts lint-style --fix --config @wordpress/stylelint-config/scss",
```

```suggestion
    "lint:css": "wp-scripts lint-style 'src/**/*.scss' --allow-empty-input",
    "fix:css": "wp-scripts lint-style 'src/**/*.scss' --fix --allow-empty-input",
```
The font css files in dist/ are causing these to fail but the linting shouldn't be running on the dist files. If we used the default folder name (build/) then wp-scripts would ignore it automatically but since we use dist/ we have to explicitly ignore it.